app/connectors/PaymentsOrchestratorConnector.scala

```diff
@@ -70,4 +71,14 @@ class PaymentsOrchestratorConnector @Inject() (
     httpClient.GET[Option[CustomerInformation]](getCustomerURL)
   }
 
+  def getDDData(vrn: Vrn)(implicit request: Request[_]): Future[Option[DirectDebitData]] = {
+
+    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))
```

Comment:
Please replace this line with:
`import req.RequestSupport._`. There is the implicit converted you were looking for :)

Reply:
And please import it on class level, so all connector methods will see it.

Reply:
Cheers